python/cugraph/cugraph/__init__.py

```diff
@@ -56,6 +56,7 @@
 
 from cugraph.centrality import (
     betweenness_centrality,
+    plc_betweenness_centrality,
```

Comment:
I didn't realize we were going to have two versions of `betweenness_centrality`.  I think this could be a problem, plus we don't want to expose a prefix like `plc_`.  Is there a reason we can't replace the current `betweenness_centrality` implementation with the new PLC-based one?

edit: @jnke2016 and I talked and he does not intend to have this merged with two BC APIs. The `plc_` version is meant to be temporary for testing, and there will be a followup commit pushed to replace the current implementation with this new one.